bin/foreman-tasks

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+foreman_root = '/usr/share/foreman'
```

Comment:
I would just double check that this path is the same as for Debian deployments. Perhaps @mmoll  can weigh in.